### PR TITLE
Modify Component - MAM

### PR DESCRIPTION
**Pull request by jkrvivian**
Add more detailed description of channel.

### CODE REVIEW

File: paper.tex [hunk 1]
```diff
@@ -103,10 +103,9 @@ \subsubsection{Broker}
...
+IOTA\cite{IOTAwhitepaper} is a feeless and scalable cryptocurrency for the IoT industry based on a the distributed ledger consists of a directed acyclic graph(DAG) of individual transactions, \textbf{Tangle}. The innovative consensus algorithm allows people to make \textbf{zero-transactions} that one could store information securely within Tangle transactions. Nevertheless, publishing encrypted streaming data to distributed ledgers is still a challenge, which one need to keep the address and session key per message, otherwise attackers could easily issue spam transactions if publishing with the same address.
```
Comment (jserv) [inline paper.tex]: Mention IOTA and MAM in the same paragraph.

Comment (jserv) [reply paper.tex]: You can say, "IOTA is ... while MAM is the second layer protocol built on top of Tangle."

Comment (jserv) [reply paper.tex]: Reworked yet?

Comment (jkrvivian) [reply paper.tex]: Still in progress.

**Comment by jkrvivian** [comment]
@jserv , I move Figure 3 (product launching) to MAM section.
Please review and check if it's smooth ><

**Pull request merged**